pkg/controller/controller.go

```diff
@@ -195,7 +199,7 @@ func (m *CertificateApprover) getKubeletCA() *x509.CertPool {
 		Namespace: configNamespace,
 		Name:      kubeletCAConfigMap,
 	}
-	if err := m.Get(context.Background(), key, configMap); err != nil {
+	if err := m.NodeClient.Get(context.Background(), key, configMap); err != nil {
```

Comment:
this should probably be called workloadClusterClient consistently with the flag and the config, is not only querying nodes.